packages/docs/src/sidebar.mdx

```diff
@@ -9,6 +9,7 @@
 - [API](/api)
 - [Theme Specification](/theme-spec)
 - [Customize](/customize)
+- [Theme UI and Gatsby](/getting-started/gatsby)
```

Comment:
Thanks! I was originally thinking this could be nested under *Getting Started*, similar to what https://mdxjs.com does. We could add similar pages for CRA, Nextjs, etc. What do you think of that approach?

Reply:
Seems perfectly reasonable to me. I wasn't sure how prominent you wanted it to be and was fine with anything. Nested under getting started is ideal! Though I'm hopeful this doc can expand beyond getting started content?

Reply:
I'm guessing we'd eventually want more Gatsby-related content in the docs -- maybe there could be an entire section dedicated to usage in Gatsby -- but seems like there'd still be value in having a getting started page for Gatsby, since using the plugin is a bit different than using the `ThemeProvider` directly

Reply:
Sounds good. We can start there and then look at additional docs later.

Reply:
Took a stab at an idea.

Reply:
Love it! Thanks! Feel free to open some more issues/PRs if you have any direction for how to start fleshing out more Gatsby-specific docs (i.e. if we want guides/sections related to usage in Gatsby themes, etc.)